composer.json

```diff
@@ -21,15 +21,19 @@
         "drupal/console": "^1.0"
     },
     "require": {
-        "composer/installers": "^1.0",
         "drupal-composer/drupal-scaffold": "^2.0.0",
         "cweagans/composer-patches": "^1.6.0",
-        "acquia/lightning": "~2.1.0"
+        "acquia/lightning": "~2.1.0",
+        "oomphinc/composer-installers-extender": "^1.1"
```

Comment:
Agree. Let's fix it now though if we can. It changes the other PRs from four changes to two.

* ~~remove composer-installers~~ (optional)
* ~~Add composer-installers-extra~~
* Add packagist
* Add new path definitions